x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/create_field/semantic_text/use_semantic_text.ts

```diff
@@ -64,7 +64,22 @@ export function useSemanticText(props: UseSemanticTextProps) {
     }
   }, [form, inferenceId, inferenceToModelIdMap]);
 
-  const handleSemanticText = (data: Field) => {
+  const isModelDownloaded = async (modelId: string) => {
```

Comment:
You probably want to wrap this in a `useCallback` for efficiency https://react.dev/reference/react/useCallback

Reply:
Updated